waltz-data/src/main/java/com/khartec/waltz/data/physical_specification_definition/PhysicalSpecDefinitionDao.java

```diff
@@ -101,9 +104,21 @@ public int delete(long specDefinitionId) {
     }
 
 
-    public int markExistingActiveAsObsolete(long specificationId) {
+    public int updateStatus(long specDefinitionId, ReleaseLifecycleStatus newStatus, String userName) {
+        return dsl.update(PHYSICAL_SPEC_DEFN)
+                .set(PHYSICAL_SPEC_DEFN.STATUS, newStatus.name())
+                .set(PHYSICAL_SPEC_DEFN.LAST_UPDATED_AT, nowUtcTimestamp())
+                .set(PHYSICAL_SPEC_DEFN.LAST_UPDATED_BY, userName)
+                .where(PHYSICAL_SPEC_DEFN.ID.eq(specDefinitionId))
```

Comment:
Perhaps an `and STATUS != newStatus` clause to prevent updating timestamp etc if no change......